validphys2/src/validphys/coredata.py

```diff
@@ -364,21 +369,21 @@ def systematic_errors(self, central_values=None):
         converted_mult_errors = self.multiplicative_errors * central_values[:, np.newaxis] / 100
         return pd.concat((self.additive_errors, converted_mult_errors), axis=1)
 
+
     def export(self, path):
         """Export the data, and error types
-        Use the same format as libNNPDF:
 
         - A DATA_<dataset>.dat file with the dataframe of accepted points
         - A systypes/STYPES_<dataset>.dat file with the error types
         """
+        from validphys.commondataparser import (
+            write_systype_to_file,
+            write_commondata_to_file,
+        )
+
```

Comment:
I disagree that they don't belong in the same place. The place I would look for write functions is the same where the read functios are. Besides, unless we have the write functions in coredata.py its not going to be trivial to avoid circular imports.

Reply:
None of the lines added to `commondataparser.py` need to import anything. You can just create a `commondatawriter.py` and then the problem is fixed. It is trivial to avoid circular imports.

> The place I would look for write functions ...

If there is a file called `commondatawriter.py` next to it I'm sure you will open the right file.

Reply:
The reader functions need to import the data definitions they are reading into. Also I don't think there is anything wrong with circular imports in this case that justifies adding files with ten lines worth of code.

Reply:
> The reader functions need to import the data definitions they are reading into.

Which reader functions? I think you are thinking of a different thing?

Reply:
I was answering to

> None of the lines added to commondataparser.py need to import anything.

saying that the functions that take files and return data structures do need to import the data structure, and there would be a circular import if you want a convenience method to read.

Reply:
Sorry, but I really cannot see which of the lines added to `commondataparser.py` is using anything imported form the outside. They are all self-contained functions.

There are no reads methods in the changes introduced by this Pull Request. I just want to have a `commondatawriter.py` with those functions.